src/common/options/rgw.yaml.in

```diff
@@ -3783,8 +3783,24 @@ options:
     other form of policies that Amazon does, so if you are mirroring
     policies between RGW and AWS, you may wish to set this to false.
   default: true
+- name: rgw_perf_counters_cache
+  type: bool
+  level: dev
+  default: false
```

Comment:
Should this behavior set to true by default? And If we're planning to keep it false as default I think we should describe more about this option, something like: Why and in what scenarios it should be used? (probably with large no. of buckets,etc..)

Reply:
No the behavior should not be enabled by default. I would not expect every single deployment to want to send metrics prometheus to prometheus via the exporter. This feature is missing docs though, which is something I will add to the PR.